tests/UserAgentsTest/FullTest.php

```diff
@@ -164,7 +164,7 @@ public function userAgentDataProvider()
                 continue;
             }
 
-            $tests = require_once $file->getPathname();
```

Comment:
I changed these to `require` instead of `require_once` because sometimes I like to run all of the UA tests in a single command, like: `./vendor/bin/phpunit -c tests/phpunit-integration.xml.dist --no-coverage --colors --verbose tests/UserAgentsTest/` and when I did this with the `require_once` the Standard and Lite tests actually would throw notices that `$tests` wasn't an array, and would end up skipping those suites since the data provider had no data to return.